docs/source/guides/writer/chapters/writing.rst

```diff
@@ -752,15 +752,15 @@ log.
 You may log something into the test logs using the methods in
 :mod:`avocado.Test.log` class attributes. Consider the example::
 
-    class output_test(Test):
+    class OutputTest(Test):
 
         def test(self):
             self.log.info('This goes to the log and it is only informational')
             self.log.warn('Oh, something unexpected, non-critical happened, '
                           'but we can continue.')
-            self.log.error('Describe the error here and don't forget to raise '
-                           'an exception yourself. Writing to self.log.error '
-                           'won't do that for you.')
+            self.log.error("Describe the error here and don't forget to raise "
+                           "an exception yourself. Writing to self.log.error "
+                           "won't do that for you.")
```

Comment:
Good catch, but I have a nitpick: let's use a single style of quotes (preferably the one enforced by Avocado's own style check.

Reply:
sure, I will update the other as well